### PR TITLE
Add time series pages

### PR DESCRIPTION
**Pull request by Shreyanand**
In this PR, I add remote source for time series pages.

There are two caveats:

1) Relative links in the main page described in this [issue](https://github.com/aicoe-aiops/time-series/issues/11).
2) Rendering plotly images on the website. Currently on my local it is not able to render notebook plotly images on the website.

### CODE REVIEW

**Comment by hemajv** [comment]
@Shreyanand do you have a preview of the changes in your [Github pages](https://github.com/operate-first/operate-first.github.io#previewing-your-changes-on-github-pages)?

File: config/data-science.yaml [hunk 1]
```diff
@@ -34,3 +34,11 @@
       href: /data-science/cloud-price-analysis/notebooks/experimental/cloud-price-analysis-EDA.ipynb
     - label: In-depth EDA for AWS data
       href: /data-science/cloud-price-analysis/notebooks/experimental/cloud-price-analysis-indepth-EDA.ipynb
+- label: Time series analysis for monitoring
+  links:
+    - label: Time series analysis for monitoring overview
+      href: /data-science/time-series/
+    - label: Fetching time series metrics data
+      href: /data-science/time-series/notebooks/ts-1-fetching-metrics.ipynb
```
Comment (hemajv) [inline config/data-science.yaml:42]: isn't the notebook file name just `fetching-metrics.ipynb` according to [here](https://github.com/aicoe-aiops/time-series/blob/master/notebooks/fetching-metrics.ipynb)?

Comment (Shreyanand) [reply config/data-science.yaml:42]: Good catch! Let's wait for this [PR](https://github.com/aicoe-aiops/time-series/pull/10) to get merged :D

Comment (durandom) [reply config/data-science.yaml:42]: @Shreyanand https://github.com/aicoe-aiops/time-series/pull/10 is merged, do you want to update here ?

**Review by durandom — CHANGES_REQUESTED** [review]

File: config/data-science.yaml (hunk 1 above)
Comment (Shreyanand) [reply config/data-science.yaml:42]: Yup! the merged PR changes the name of the file to match this.

**Comment by Shreyanand** [comment]
> @Shreyanand do you have a preview of the changes in your [Github pages](https://github.com/operate-first/operate-first.github.io#previewing-your-changes-on-github-pages)?

@hemajv It is working fine on my local but when I run make gh-pages-fork, the content published on https://shreyanand.github.io/operate-first.github.io has the side navigation links missing. I'm not sure why that's happening because the changes in this PR seem ok and it works fine on my local.

**Comment by hemajv** [comment]
> > @Shreyanand do you have a preview of the changes in your [Github pages](https://github.com/operate-first/operate-first.github.io#previewing-your-changes-on-github-pages)?
> 
> @hemajv It is working fine on my local but when I run make gh-pages-fork, the content published on https://shreyanand.github.io/operate-first.github.io has the side navigation links missing. I'm not sure why that's happening because the changes in this PR seem ok and it works fine on my local.

ah yes, I think @aakankshaduggal had the same issue as well. Seems like sometimes the git pages preview is broken. I do see your changes at https://shreyanand.github.io/operate-first.github.io/data-science/time-series/ though.

**Comment by SankBad** [comment]
I also had the same issue a week back.

**Review by hemajv — APPROVED** [review]
lgtm :ship:

**Review by tumido — APPROVED** [review]

**Pull request merged**